a3p-integration/proposals/a:vaults-auctions/agd-tools.js

```diff
@@ -201,3 +201,18 @@ export const getProvisionPoolMetrics = async () => {
   const path = `published.provisionPool.metrics`;
   return getQuoteBody(path);
 };
+
+export const getAuctionInstance = async price => {
```

Comment:
not really an "agd tool". Also would be more general with a little tweak:
```suggestion
export const getInstanceBoardId = async instanceName => {
```
(`price` isn't used)

Not a blocker for this repo but in synthetic-chain package it should be designed for re-use.